common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java

```diff
@@ -122,6 +124,7 @@ private ScheduledFuture<?> scheduledSneakPress(String msg) {
     @SubscribeEvent
     public void onWorldStateChange(WorldStateEvent e) {
         currentDialogue = null;
+        currentlyBlocking = true;
```

Comment:
You do not need to set this here. `currentlyBlocking` is only checked in `renderDialogue`, and that is only called if we have a `currentDialgue`, and that is set at the same time as you set `currentlyBlocking` in the event handler. So this is just confusing. (Especially as you set it to true)